x-pack/plugins/apm/public/components/app/Settings/agent_keys/create_agent_key/agent_key_callout.tsx

```diff
@@ -66,9 +67,8 @@ export function AgentKeyCallOut({ name, token }: Props) {
             </EuiCopy>
           }
         >
-          <input
-            type="text"
-            className="euiFieldText euiFieldText--inGroup"
+          <EuiFieldText
+            className="euiFieldText--inGroup"
```

Comment:
Hmm, maybe it's because we are using `EuiFormControlLayout`. The docs say:

> EuiFormControlLayout is generally used internally to consistently style form controls, but it’s published in case you want to create your own form control which matches those of EUI. The examples below demonstrate its various states.

I think it's better to avoid using this - even if we lose a bit of control. We haven't needed it before.

Reply:
A very similar component is being used [here](https://github.com/elastic/kibana/blob/48d18464c8cd7fccfc6192dd14127d7312aaa22c/x-pack/plugins/security/public/components/token_field.tsx#L53) in API key management in security plugin and tried to keep the design. Can we have a transparent background when using just the field? Otherwise, this is how it looks like (which doesn't seem that bad to me tbh)
![image](https://user-images.githubusercontent.com/5831975/145260039-261fe5f5-ac18-4571-bf67-9dfdeb017a09.png)

Reply:
This is with `EuiFormControlLayout`
![image](https://user-images.githubusercontent.com/5831975/145260218-7917688e-24bf-4bc3-b0e4-fbeabc8af2ca.png)

Reply:
> A very similar component is being used [here](https://github.com/elastic/kibana/blob/48d18464c8cd7fccfc6192dd14127d7312aaa22c/x-pack/plugins/security/public/components/token_field.tsx#L53) in API key management in security plugin and tried to keep the design. Can we have a transparent background when using just the field? Otherwise, this is how it looks like (which doesn't seem that bad to me tbh)
> 
> ![image](https://user-images.githubusercontent.com/5831975/145260039-261fe5f5-ac18-4571-bf67-9dfdeb017a09.png)
> 
>  

I think I prefer this tbh

Reply:
Thanks, changed it which makes the component simpler 💯